scripts/build_all.sh

```diff
@@ -206,8 +231,7 @@ function ffmpeg_install() {
     make \
         -C "$BUILD_DIR" \
         $MAKE_ARGS_STRING \
-        install \
-        V=1
+        install VERBOSE=1
```

Comment:
This btw isn't valid, I believe autotools-generated makefiles support `V`, but not `VERBOSE`.